guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,47 @@
 {
```

Comment:
<p>@jyaistmap the release only had the following bugfixes in the comment below. Can you please update this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>